src/editor/EditorStatusBar.js

```diff
@@ -226,6 +227,25 @@ define(function (require, exports, module) {
     }
 
     /**
+     * Update LineEndings label
+     * @param {Event} event (unused)
+     * @param {Editor} editor Current editor
+     * @param {boolean=} doNotAnimate True if state should not be animated
+     */
+    function _updateLineEndingsLabel(event, editor, doNotAnimate) {
+        /* Toggle text */
+        if($statusLineEndings.text() === Strings.STATUSBAR_LINE_ENDINGS_CRLF) {
```

Comment:
This line fails JSLint.  Please make sure all your files have _zero_ JSLint warnings.